agent/engine/docker_task_engine.go

```diff
@@ -1023,6 +1023,16 @@ func (engine *DockerTaskEngine) pullAndUpdateContainerReference(task *apitask.Ta
 	return metadata
 }
 
+func (engine *DockerTaskEngine) getPullImageIamCredentials(task *apitask.Task, container *apicontainer.Container) (credentials.IAMRoleCredentials, bool, string) {
```

Comment:
nit: can we rename the method to `getPullImageIAMCredentials` ?

Reply:
sure